src/DynamoCoreWpf/Views/Preview/Watch3DView.xaml.cs

```diff
@@ -1180,7 +1202,7 @@ private void AggregateRenderPackages(PackageAggregationParams parameters)
                 }
                 else
                 {
-                    meshGeometry3D = new DynamoGeometryModel3D()
+                    meshGeometry3D = new DynamoGeometryModel3D(renderTechnique)
```

Comment:
The `DynamoGeometryModel3D` type now takes the `RenderTechnique` as a parameter to the constructor. Previously, in Helix, these render techniques were stored in a static dictionary.